WcaOnRails/app/models/concerns/localized_sortable.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Appends a bunch of methods to *Cachable* classes to expose
+# a collection of sorted objects based on the user's locale.
+require 'active_support/concern'
+
+module LocalizedSortable
```

Comment:
It says nothing about the real/fictive stuff going on here, but honestly I don't have any appropriate name on mind, so I'm fine with it ^^

Reply:
I know, I struggled to find an "appropriate" name :(